enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -259,6 +271,24 @@ graphroot = "/var/lib/containers/storage"
 	}, nil
 }
 
+func getPodmanVersion(ctx context.Context) (*semver.Version, error) {
+	res := runPodman(ctx, "version", nil /*=stdio*/)
```

Comment:
If you add `--format '{{.Client.Version}}'` it'll print just the client version and nothing else, which will save a little bit of code in this function, and it looks like that was added pre-1.0 so should work for all except the most out-of-date podman versions.

Reply:
Nice! much cleaner.